keyrings/alt/pyfs.py

```diff
@@ -9,14 +9,44 @@
 from keyring.backend import KeyringBackend, NullCrypter
 from . import keyczar
 
+# support getting module version
+import pkg_resources
+
+# support parsing version string
+from packaging import version
+
+# check FS version
 try:
-    import fs.opener
-    import fs.osfs
-    import fs.errors
-    import fs.path
-    import fs.remote
+    FS_VER = version.parse(pkg_resources.get_distribution("fs").version)
+except pkg_resources.DistributionNotFound:
+    raise ImportError
+
+import fs.osfs
+import fs.errors
+import fs.path
+
+try:
+    if FS_VER < version.parse("2.0.0"):
+        import fs.remote
+    else:
+        import fs.wrap
 except ImportError:
-    pass
```

Comment:
Previously, the behavior was not to raise an error when fs wasn't available. This new approach causes an ImportError when fs isn't available. That means that any users without fs installed will get an error when importing keyring. For example, I applied this diff.

```diff
diff --git a/keyrings/alt/pyfs.py b/keyrings/alt/pyfs.py
index 08addd2..cab6713 100644
--- a/keyrings/alt/pyfs.py
+++ b/keyrings/alt/pyfs.py
@@ -17,6 +17,7 @@ try:
     import fs.remote
 except ImportError:
     pass
+import doesnotexist
 
 
 def has_pyfs():
```

Then:

```
keyrings.alt master $ .tox/python/bin/python -c "import keyring"
Error initializing plugin EntryPoint(name='pyfs', value='keyrings.alt.pyfs', group='keyring.backends').
Traceback (most recent call last):
  File "/Users/jaraco/code/main/keyrings.alt/.tox/python/lib/python3.8/site-packages/keyring/backend.py", line 198, in _load_plugins
    init_func = ep.load()
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/importlib/metadata.py", line 77, in load
    module = import_module(match.group('module'))
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/jaraco/code/main/keyrings.alt/keyrings/alt/pyfs.py", line 20, in <module>
    import doesnotexist
ModuleNotFoundError: No module named 'doesnotexist'
```

These modules are designed to be lenient to errors and to signal their viability or not through the 'priority' method.